### PR TITLE
fix don't report unactionable w3name request errors to Sentry

### PR DESCRIPTION
**Pull request by juliangruber**
Ref https://filecoinproject.slack.com/archives/C06SFMDMUAU/p1734275910797789
Related: https://github.com/filecoin-station/core/issues/651

### CODE REVIEW

**Review by pyropy — APPROVED** [review]
Changes look good! I wonder if we'd have any benefits of keeping metrics on how often are these errors happening?

**Comment by bajtos** [comment]
> Changes look good! I wonder if we'd have any benefits of keeping metrics on how often are these errors happening?

That's what I was thinking too!

**Review by bajtos — APPROVED** [review]

File: lib/telemetry.js [hunk 1]
```diff
@@ -102,3 +102,10 @@ export const runMachinesLoop = async ({ STATION_ID }) => {
     await timers.setTimeout(24 * 3600 * 1000) // 1 day
   }
 }
+
+export const reportW3NameError = () => {
+  const point = new Point('w3name-error')
```
Comment (pyropy) [inline lib/telemetry.js:107]: Nice 🎉

**Review by pyropy — COMMENTED** [review]
Looks good to me :shipit:

**Comment by juliangruber** [comment]
I hadn't yet set up commit signing on this machine, all future commits will be signed

**Comment by juliangruber** [comment]
Also added to Grafana

**Pull request merged**